src/httpserver/http_utils.hpp

```diff
@@ -301,6 +304,13 @@ class arg_comparator {
       * @param first string
       * @param second string
      **/
+     bool operator()(std::string_view x, std::string_view y) const {
+#ifdef CASE_INSENSITIVE
+         COMPARATOR(x, y, std::toupper);
+#else
+         COMPARATOR(x, y,);  // NOLINT(whitespace/comma)
+#endif
+     }
      bool operator()(const std::string& x, const std::string& y) const {
```

Comment:
Given we have the "string_view" version, I'd have the "string" version just use the string view one (which saves an ifdef/else block)

Reply:
Good idea :+1: